src/ddspdrum/torchmodule.py

```diff
@@ -12,7 +12,7 @@
 
 from ddspdrum.defaults import SAMPLE_RATE
 from ddspdrum.modparameter import ModParameter
-from ddspdrum.torchutil import midi_to_hz, reverse_signal
+from ddspdrum.torchutil import midi_to_hz, reverse_signal, linspace
```

Comment:
Linter is complaining about the ordering of imports, wants them to be alphabetical

Reply:
There's something a little S&M about this linting workflow.